### PR TITLE
Publicise community managers team

### PR DESCRIPTION
**Pull request by clementbiron**
Hey @emmanuellar and @Cli4d, we could like to make you appear to a dedicated section of [the about page](https://opentermsarchive.org/about) on the the Open Terms Archive website.

If you agree, is the picture, the name and the link I put in this PR suitable for you?

image

### CODE REVIEW

File: content/pages/en/about.mdx [hunk 1]
```diff
@@ -53,6 +53,12 @@ grid.gutters: 8
 
   <p className="text__light mb__3XL">This project was in part [funded](https://nlnet.nl/project/TOSDR-OTA/) through the [NGI0 Entrust](https://nlnet.nl/entrust) Fund, a fund established by [NLnet](https://nlnet.nl/) with financial support from the European Commission's [Next Generation Internet](https://ngi.eu/) programme, under the aegis of DG Communications Networks, Content and Technology under grant agreement N°101069594.</p>
 
+  ### Community managers
```
Comment (MattiSG) [inline content/pages/en/about.mdx:56]: > In English, title capitalisation is with an uppercase on each word… don't ask me why.
```suggestion
  ### Community Managers
```

**Review by MattiSG — APPROVED** [review]
Thanks @clementbiron!

**Comment by emmanuellar** [comment]
The picture and name look perfect. Thank you Clement.

**Comment by Cli4d** [comment]
The picture, name, and link look good. Thank you so much @clementbiron, @MattiSG, and @Ndpnt for this gesture. I feel appreciated by the team, as it caught me off guard. I wasn't expecting such a kind act.

**Review by Cli4d — APPROVED** [review]
It all looks good to me

**Review by emmanuellar — APPROVED** [review]
This looks great. Thanks for the addition.

**Pull request merged**